subprojects/platform-play/src/test/groovy/org/gradle/play/internal/PlayPlatformResolverTest.groovy

```diff
@@ -78,6 +78,34 @@ class PlayPlatformResolverTest extends Specification {
         requirement << ["play-2.3.4", [play: '2.3.4']]
     }
 
+    def "resolves platform for Play 2.4.x"() {
+        when:
+        def playPlatform = resolve(requirement)
+
+        then:
+        playPlatform.name == "play-2.4.6"
+        playPlatform.playVersion == "2.4.6"
+        playPlatform.javaPlatform.targetCompatibility == JavaVersion.current()
+        playPlatform.scalaPlatform.scalaVersion == "2.11.4"
+
+        where:
+        requirement << ["play-2.4.6", [play: '2.4.6']]
+    }
+
+    def "resolves platform for Play 2.5.x"() {
+        when:
+        def playPlatform = resolve(requirement)
+
+        then:
+        playPlatform.name == "play-2.5.4"
+        playPlatform.playVersion == "2.5.4"
+        playPlatform.javaPlatform.targetCompatibility == JavaVersion.current()
+        playPlatform.scalaPlatform.scalaVersion == "2.11.4"
+
+        where:
+        requirement << ["play-2.5.4", [play: '2.5.4']]
+    }
```

Comment:
Please collapse these to tests into one. You should be able to do that by adjusting the where class.